cruise-control-client/setup.py

```diff
@@ -11,7 +11,7 @@
 
 setuptools.setup(
     name='cruise-control-client',
-    version='1.0.2',
+    version='1.0.3',
```

Comment:
Nit: Is it possible to add a documentation on when to bump up client version?

Reply:
> Thanks for the patch -- @mgrubent I see some naming conflicts between this patch and #1035. Should this patch be prioritized over the other one?

Let's merge this one first then I'll work with @kidkun to merge in #1035